src/main/java/John.java

```diff
@@ -0,0 +1,161 @@
+import commands.Deadline;
+import commands.Event;
+import commands.Task;
+import commands.Todo;
+
+
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.List;
+
+public class John {
+    //Load in the data and import it into the task list
+    static List<Task> tasks;
+    static {
+        try {
+            tasks = DataManager.readData("src/main/java/data.txt");
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static void main(String[] args) {
+        System.out.println("ヽ(•‿•)ノ");
+         System.out.println("-----------------------------------");
+         System.out.println("Heya, Im ya Chatbot, call me John!\nWhat can I do for you today?");
+         // Taking input from the user
+         getUserInput();
+         System.out.println("\n---------------------------------");
+         System.out.println("Hope to see you soon Bruda.");
+    }
+
+    // Method to take input and return it
+    public static void getUserInput() {
```

Comment:
method is very long, more than 30 lines of code. Try to apply abstraction for each command